WooCommerce/Classes/ViewRelated/Products/ProductsViewController.swift

```diff
@@ -694,6 +704,8 @@ private extension ProductsViewController {
         }
         displayEmptyStateViewController(emptyStateViewController)
         emptyStateViewController.configure(config)
+        // Hide toolbar when user doesn't have products
+        toolbar.isHidden = true
```

Comment:
Could you explain to me why is this needed? I would think that the change introduced above was enough! Maybe I'm missing something. 😅

Reply:
Yep, it's needed because we're using the functions `onDidChangeContent` and `onDidResetContent`, but if a user doesn't have products since the first time that opened the view, we need this to remove the toolbar